packages/odyssey-react-mui/src/themes/odyssey/typography.ts

```diff
@@ -24,82 +24,103 @@ export const typography: ThemeOptions["typography"] = {
-    fontWeight: "600",
+    fontWeight: 600,
     fontSize: "2.286rem",
     lineHeight: "1.28571429",
     letterSpacing: "initial",
+    marginBottom: "0.57142857rem",
   },
   h3: {
     fontFamily:
       "'Public Sans', '-apple-system', 'BlinkMacSystemFont', 'Segoe UI', 'Roboto', 'Oxygen-Sans', 'Ubuntu', 'Cantarell', 'Helvetica Neue', 'Noto Sans Arabic', sans-serif",
-    fontWeight: "600",
+    fontWeight: 600,
     fontSize: "2rem",
     lineHeight: "1.28",
     letterSpacing: "initial",
+    marginBottom: "0.57142857rem",
   },
   h4: {
     fontFamily:
       "'Public Sans', '-apple-system', 'BlinkMacSystemFont', 'Segoe UI', 'Roboto', 'Oxygen-Sans', 'Ubuntu', 'Cantarell', 'Helvetica Neue', 'Noto Sans Arabic', sans-serif",
-    fontWeight: "600",
+    fontWeight: 600,
     fontSize: "1.571rem",
     lineHeight: "1.27272727",
     letterSpacing: "initial",
+    marginBottom: "0.57142857rem",
   },
   h5: {
     fontFamily:
       "'Public Sans', '-apple-system', 'BlinkMacSystemFont', 'Segoe UI', 'Roboto', 'Oxygen-Sans', 'Ubuntu', 'Cantarell', 'Helvetica Neue', 'Noto Sans Arabic', sans-serif",
-    fontWeight: "600",
+    fontWeight: 600,
     fontSize: "1.286rem",
     lineHeight: "1.333333333",
     letterSpacing: "initial",
+    marginBottom: "0.57142857rem",
   },
   h6: {
     fontFamily:
       "'Public Sans', '-apple-system', 'BlinkMacSystemFont', 'Segoe UI', 'Roboto', 'Oxygen-Sans', 'Ubuntu', 'Cantarell', 'Helvetica Neue', 'Noto Sans Arabic', sans-serif",
-    fontWeight: "600",
+    fontWeight: 600,
     fontSize: "1.143rem",
     lineHeight: "1.25",
     letterSpacing: "initial",
+    marginBottom: "0.57142857rem",
   },
-  //subtitle1: undefined,
-  //subtitle2: undefined,
-  body1: {
+  subtitle1: undefined,
+  subtitle2: undefined,
+  body: {
     fontFamily:
       "'Public Sans', '-apple-system', 'BlinkMacSystemFont', 'Segoe UI', 'Roboto', 'Oxygen-Sans', 'Ubuntu', 'Cantarell', 'Helvetica Neue', 'Noto Sans Arabic', sans-serif",
-    fontWeight: "400",
+    fontWeight: 400,
     fontSize: "1rem",
     lineHeight: "1.42857143",
     letterSpacing: "initial",
   },
-  //body2: undefined,
-  //button: undefined,
+  body1: undefined,
+  body2: undefined,
+  button: undefined,
   caption: {
     fontFamily:
       "'Public Sans', '-apple-system', 'BlinkMacSystemFont', 'Segoe UI', 'Roboto', 'Oxygen-Sans', 'Ubuntu', 'Cantarell', 'Helvetica Neue', 'Noto Sans Arabic', sans-serif",
-    fontWeight: "400",
+    fontWeight: 400,
     fontSize: "0.875rem",
     lineHeight: "1.42857143",
     letterSpacing: "initial",
   },
-  //overline: undefined,
+  overline: undefined,
 };
 
 // Update the Typography's variant prop options
 // Unsure where we want to store this
-//declare module "@mui/material/Typography" {
-//interface TypographyPropsVariantOverrides {
-//subtitle1: false;
-//subtitle2: false;
-//body2: false;
-//button: false;
-//overline: false;
-//}
-//}
+declare module "@mui/material/styles" {
```

Comment:
@sherizada-okta I'm not sure where these should live. I have them alongside the theme overrides right now for ease of comparison, but we should be exporting them somewhere.

Reply:
Looks like the `declare`s should be in separate files following the pattern `[filename].d.ts` (https://www.typescriptlang.org/docs/handbook/declaration-files/introduction.html). 
For now we can have an `index.d.ts` housing them.

Reply:
And then inside packages/odyssey-react-mui/tsconfig.json we'll need to add:
```
  "include": [
      "@okta/odyssey-react-mui/themes/odyssey/types.d.ts"
    ]
```

Reply:
I think we get the `createTheme` error because the theme object can't be parsed without the type augmentations:

```src/themes/odyssey/typography.ts(80,3): error TS2322: Type '{ htmlFontSize: number; fontFamily: "'Public Sans', '-apple-system', 'BlinkMacSystemFont', 'Segoe UI', 'Roboto', 'Oxygen-Sans', 'Ubuntu', 'Cantarell', 'Helvetica Neue', 'Noto Sans Arabic', sans-serif"; ... 18 more ...; overline: undefined; }' is not assignable to type 'TypographyOptions | ((palette: Palette) => TypographyOptions) | undefined'.
  Object literal may only specify known properties, and 'body' does not exist in type 'TypographyOptions | ((palette: Palette) => TypographyOptions)'```

This occurs when `types.d.ts` is included in our `tsconfig.json` as well -- but not when it lives alongside the theme object, which is odd.